### PR TITLE
Issue #25: System / Enhance Filesystem Utilization

### LINKED ISSUE #25: System / Enhance Filesystem Utilization
## Description

Currently the Filesystem Usage is reported based on the PromQL
```promql
system_filesystem_utilization_ratio{host_name="$host_name",system_filesystem_state="used"}
```
Since the Linux connector reports the reserved usage we should update the PromQL to:
```promql
sum(system_filesystem_utilization_ratio{host_name="$host_name",system_filesystem_state=~"used|reserved"}) by (id)
```

## Additional updates

- Add table legends to the network, filesystem utilization and disk activity graphs, display the last* value for each legend 
- Left align the filesystem path in the Filesystem tables. (There are two summary + details).
- Update the unit in the legend of the disk activity graph.
- Remove extra whitespace from the PromQL used to report the disk activity graph.

### PR DESCRIPTION
**Pull request by NassimBtk**
- Enhanced the PromQL query to include the filesystem utilization percentage.
- Added table legends to the network, filesystem utilization and disk activity graphs and displayed the last* value for each legend.
- Left aligned filesystem paths in the Filesystem tables.
- Updated unit in the legend of the disk activity graph.
- Removed extra whitespace from the PromQL used to report the disk activity graph.

-------------------------------
### Testing
image
image
image

### CODE REVIEW

**Comment by NassimBtk** [comment]
Thank you @MarcDouguedroit for your help.

File: src/dashboards/System/System (MetricsHub).json [hunk 1]
```diff
@@ -2839,7 +2877,7 @@
       "type": "table"
     }
   ],
-  "refresh": "1m",
+  "refresh": "5s",
```
Comment (bertysentry) [inline src/dashboards/System/System (MetricsHub).json]: This is probably way to short! 😅

Comment (NassimBtk) [inline src/dashboards/System/System (MetricsHub).json]:
```suggestion
  "refresh": "30s",
 ```

Comment (NassimBtk) [reply src/dashboards/System/System (MetricsHub).json]: We set the refresh to 30s to be consistent with 'MetricsHub Main' 😉

**Review by MarcDouguedroit — APPROVED** [review]

**Pull request merged**